variables.tf

```diff
@@ -18,6 +18,18 @@ variable "create_sns_topic" {
   default = false
 }
 
+variable "sns_encryption_key_id" {
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK."
+  type        = string
+  default     = "" # Use "alias/aws/sns" for AWS Managed Key
+}
+
+variable "sqs_queue_kms_master_key_id" {
+  type        = string
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SQS Queue or a custom CMK"
+  default     = "" # Use "alias/aws/sqs" for AWS Managed Key
```

Comment:
Why can't we keep the defaults of the upstream module here ?

Reply:
cc: @max-lobur

Reply:
I didn't want to change expected behavior. I don't have SNS listeners on my topic so I actually stopped creating it, which solves our encryption need.